pysat/utils/stats.py

```diff
@@ -9,21 +9,40 @@
 import numpy as np
 
 
-# determine the median in 1 dimension
 def median1D(self, bin_params, bin_label, data_label):
+    """Calculates the median for a series of binned data.
+
+    ** Will be remvoed in a future version now that ssnl.avgs has a
```

Comment:
A deprecation warning in the code would also be good. Provides a more active announcement of the change.

Reply:
added.